js/script.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+    "use strict"
+    
+    document.getElementById("btn-hamburguer").onclick = function(){
+        let pages = document.querySelector(".paginas")
+        let hasAppear = pages.style.visibility;
+        
+        if(screen.width > 800) pages.style.visibility = "visible";
+        else if(hasAppear == "visible") pages.style.visibility = "collapse";

```

Comment:
Codacy has a fix for the issue: [Replace `········else·if·(hasAppear·==·"visible")·pages.style.visibility·=·"collapse";␍` with `····else·if·(hasAppear·==·"visible")·pages.style.visibility·=·"collapse";`](https://app.codacy.com/gh/Jeffer48/Consultarq/pullRequest?prid=9456242)
```suggestion
    else if (hasAppear == "visible") pages.style.visibility = "collapse";
```